sql/006. ALTER Components ADD ImageUrl.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE Gallery.Components
+ADD ImageUrl VARCHAR(255);
```

Comment:
Typy komponentów spoko ale ImageUrl nie będziemy dodawać, już to jest inaczej zrobione - url jest budowany na podstawie identyfikatora komponentu (ID)

Reply:
nieważne, to służy do czegos innego. moj blad